ggml-cuda.cu

```diff
@@ -7388,14 +7458,24 @@ static void soft_max_f32_cuda(const float * x, const float * y, float * dst, con
     }
 }
 
+static void im2col_f32_f32_cuda(const float* x, float* dst,
```

Comment:
This function should also be a template instead.

Reply:
fixed in c29a855453dd29f5a1cb8bc3edeeda2ae50fe460